Docs/DafnyRef/DafnyRef.mdk

```diff
@@ -3727,34 +3755,83 @@ Note that when there is no body, Dafny assumes that the *ensures*
 clauses are true without proof.
 
 ### Constructors
-To write structured object-oriented programs, one often relies on that
-objects are constructed only in certain ways.  For this purpose, Dafny
+To write structured object-oriented programs, one often relies on 
+objects being constructed only in certain ways.  For this purpose, Dafny
 provides _constructor (method)s_, which are a restricted form of
 initialization methods.  A constructor is declared with the keyword
-`constructor` instead of `method`.  When a class contains a
-constructor, every call to `new` for that class must be accompanied
-with a call to one of the constructors.  Moreover, a constructor
+`constructor` instead of `method`.  
+Every call to `new` for a class must be accompanied
+by a call to one of its constructors.  Moreover, a constructor
```

Comment:
The previous text is correct, the new text is not. A call to `new` must be accompanied by a call to one of the class's constructors only if the class declares any constructors at all.